kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -31,25 +31,52 @@
 import org.kitesdk.data.spi.Registration;
 
 /**
- * <p>Convenience methods for working with {@link Dataset} instances.</p>
- *
+ * Methods for working with {@link Dataset} instances.
+ * <p>
+ * <b>URIs</b><br />
+ * All methods require a URI that identifies a dataset, view, or
+ * repository. The URI must begin with the scheme {@code dataset:}, 
+ * {@code view:}, or {@code repo:}. The remainder of the URI is 
+ * implementation specific, depending on the dataset scheme.
+ * <p>
+ * For example, the URI {@code dataset:hive:/movies/ratings} 
+ * references a Hive dataset named <i>ratings</i> in the
+ * <i>movies</i> namespace.
+ * <p>
+ * The URI {@code view:hdfs:/user/myfiles/movies/ratings?year=2015&month=3}
+ * references a view of an HDFS dataset named <i>ratings</i> in the
+ * <i>movies</i> namespace in the {@code /user/myfiles} path. The view 
+ * is filtered to include records from only March, 2015.
+ * <p>
+ * See <a href="http://kitesdk.org/docs/current/URIs.html">Dataset and View
+ * URIs</a>.
+ * <p>
+ * <b>Entities</b><br />
+ * <i>Entities</i> are analagous to <i>records</i> in database terminology.
+ * The term is used in the API to emphasize that an entity can include not
+ * only primitive objects, but also complex objects such as hash maps. 
+ *  
  * @since 0.8.0
  */
 public class Datasets {
 
   /**
    * Load a {@link Dataset} or {@link View} for the given {@link URI}.
+   * {@code load} verifies that the dataset exists, retrieves the dataset's
+   * metadata, and verifies that you can communicate with its services.
    * <p>
-   * If the URI is a dataset URI, the unfiltered Dataset will be returned.
+   * URIs must begin with {@code dataset:} or {@code view:}. The remainder of
+   * the URI is implementation specfic, depending on the dataset scheme.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * If you use a dataset URI, {@code load} returns the unfiltered dataset.
+   * If you use a view URI, {@code load} returns a subset of the data, based on 
```

Comment:
Changed it. Also saying subset of the dataset, rather than the data, to make it parallel. I made the change in all 4 flavors of the load method.
***
   * If you use a dataset URI, {@code load} returns the unfiltered dataset.
   * If you use a view URI, {@code load} returns a {@code View} configured to read 
   * a subset of the dataset.